README.md

```diff
@@ -0,0 +1,70 @@
+<div align="center">
+  <img src="https://github.com/hmartiins/chat/assets/51277667/fca09cd1-d66a-4843-a6f4-cbbc1bfd738f" width="200" />
+
+  <h1>Gotter Chat</h1>
+  <p>💬 Application for exchanging messages in real time 📱</p>
+</div>
+
+## Project Overview
+
+This project is a backend implementation in Go for a real-time chat system using Socket.IO. The primary goal is to create a scalable and efficient server that facilitates communication between multiple clients in real-time.
```

Comment:
What do you think of us using the term "WebSocket" instead of "Socket.IO"? Socket.IO, I believe, refers to a javascript library, and not to the type of communication we are going to use

```suggestion
This project is a backend implementation in Go for a real-time chat system using WebSocket. The primary goal is to create a scalable and efficient server that facilitates communication between multiple clients in real-time.
```